sdks/python/apache_beam/transforms/periodicsequence.py

```diff
@@ -50,6 +50,10 @@ def create_tracker(self, restriction):
   def restriction_size(self, unused_element, restriction):
     return restriction.size()
 
+  # On drain, immediately stop emitting new elements
```

Comment:
Pls consider adding a unit test.

Reply:
I thought about it, but it seems like more trouble/flake risk than its worth. Testing this effectively would pretty much need to be an integration test on dataflow, and right now we don't have that capability (triggering a drain) built into our test framework. We could add a unit test that truncate always returns none on this restriction, but that seems like overkill (and goes beyond any of the testing we generally apply to truncate or restrictions generally

Reply:
Ack. Thanks.